src/sync/sync_user.hpp

```diff
@@ -64,18 +74,21 @@ friend class SyncSession;
     // Log the user out and mark it as such. This will also close its associated Sessions.
     void log_out();
 
-    // Whether the user was configured as an 'admin user' (directly uses its user token
-    // to open Realms).
-    bool is_admin() const
+    // Whether the user has administrator privileges, and howso.
+    SyncUserAdminMode admin_mode() const
     {
-        return m_is_admin;
+        return m_admin_mode;
     }
 
     std::string identity() const
     {
         return m_identity;
     }
 
+    // A flag which can be set by a binding to indicate whether the Realm Object Server
+    // indicates that this user has server administration privileges.
+    bool marked_as_object_server_admin;
```

Comment:
No, that looks like cruft left over from the original version of the design. I'll remove it and make sure nothing else like it is still hanging around.